src/components/filter/FilterItem.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Checkbox from 'material-ui/Checkbox';
+import capitalize from '../../util';
+
+
+const FilterItem = React.createClass({
+  getInitialState() {
+    return {
+      checked: false
+    };
+  },
+  onCheck() {
+    const checked = this.state.checked;
+    this.setState({checked: !checked});
+    this.props.onCheck(this.props.tag, !checked);
+  },
+  render(){
+    const tag = capitalize(this.props.tag);
+    return (
+      <Checkbox
+        label={tag}
+        checked={this.state.checked}
+        onCheck={this.onCheck}
```

Comment:
Hm, jeg ser at det er FilterItem som har staten for om et filter er checked eller ei, men at onCheck er sendt ned fra FrontPage (handleOnCheck) via flere lag med komponenter for å gi beskjed opp til FrontPage slik at den kan displaye riktige kurs. Allerede her begynner det å føles riktig å innføre Redux... men kan jo tenke litt på om det er en enklere måte å gjøre det på. Men en ting om gangen. Hvis vi skal legge inn Redux, gjør vi det i en egen pullrequest senere. La oss fikse de andre tingene først :)